### PR TITLE
Implement 2 ULDUM cards

### PR DESCRIPTION
**Pull request by doobee98**
This revision inculdes:

- Implement 2 ULDUM cards
  - Pharaoh Cat (ULD_186)
  - Whirlkick Master (ULD_231)

### CODE REVIEW

File: Includes/Rosetta/PlayMode/Conditions/SelfCondition.hpp [hunk 1]
```diff
@@ -203,6 +203,10 @@ class SelfCondition
     //! \return Generated SelfCondition for intended purpose.
     static SelfCondition IsOverloadCard();
 
+    //! SelfCondition wrapper for checking the card has combo.
+    //! \return Generated SelfCondition for intended purpose.
+    static SelfCondition IsComboCard();
```
Comment (utilForever) [inline Includes/Rosetta/PlayMode/Conditions/SelfCondition.hpp:208]: I suggest the name of this method to `HasCombo()` due to it has more readability.
What do you think?

File: Sources/Rosetta/PlayMode/Conditions/SelfCondition.cpp [hunk 2]
```diff
@@ -420,6 +420,18 @@ SelfCondition SelfCondition::IsOverloadCard()
     });
 }
 
+SelfCondition SelfCondition::IsComboCard()
+{
+    return SelfCondition([=](Playable* playable) -> bool {
+        if (playable->GetGameTag(GameTag::COMBO) >= 1)
```
Comment (utilForever) [inline Sources/Rosetta/PlayMode/Conditions/SelfCondition.cpp:426]: The return value of method `GetGameTag(GameTag::COMBO)` is 0 or 1.
So, I suggest change the code to`return playable->GetGameTag(GameTag::COMBO) == 1;`.

Comment (doobee98) [reply Sources/Rosetta/PlayMode/Conditions/SelfCondition.cpp:426]: Good. Sorry for having not checked it.

File: Includes/Rosetta/PlayMode/Conditions/SelfCondition.hpp (hunk 1 above)
Comment (doobee98) [reply Includes/Rosetta/PlayMode/Conditions/SelfCondition.hpp:208]: Other methods like it are
`IsBattlecryCard()`, `HasDeathrattle()`, `HasReborn()`, `IsOverloadCard()`, etc...

So I think, if an ability is for the time of playing a card, `IsAbilityCard()` can be better.
And, if an ability is for the minions already played (in field), `HasAbility()` can be better.

The ability 'Combo' is the former case. So I choose `IsComboCard()`.

What do you think about it?

Comment (utilForever) [reply Includes/Rosetta/PlayMode/Conditions/SelfCondition.hpp:208]: Agree. Thanks for your opinion. 😄

File: Sources/Rosetta/PlayMode/Conditions/SelfCondition.cpp (hunk 2 above)
Comment (utilForever) [reply Sources/Rosetta/PlayMode/Conditions/SelfCondition.cpp:426]: Could you change your code by applying it?

Comment (doobee98) [reply Sources/Rosetta/PlayMode/Conditions/SelfCondition.cpp]: I do it :D

**Pull request merged**